2003-02-07  Jakub Jelinek  <jakub@redhat.com>

	* elf32-s390.c (elf_s390_size_dynamic_sections): Set relocs to TRUE
	even if there is just non-empty .rela.plt.

--- bfd/elf32-s390.c.jj	2003-01-24 18:15:38.000000000 -0500
+++ bfd/elf32-s390.c	2003-02-07 16:48:18.000000000 -0500
@@ -1,5 +1,5 @@
 /* IBM S/390-specific support for 32-bit ELF
-   Copyright 2000, 2001, 2002 Free Software Foundation, Inc.
+   Copyright 2000, 2001, 2002, 2003 Free Software Foundation, Inc.
    Contributed by Carl B. Pedersen and Martin Schwidefsky.
 
    This file is part of BFD, the Binary File Descriptor library.
@@ -2014,7 +2014,7 @@ elf_s390_size_dynamic_sections (output_b
 	}
       else if (strncmp (bfd_get_section_name (dynobj, s), ".rela", 5) == 0)
 	{
-	  if (s->_raw_size != 0 && s != htab->srelplt)
+	  if (s->_raw_size != 0)
 	    relocs = TRUE;
 
 	  /* We use the reloc_count field as a counter if we need

